src/qibo/validation/gate_set_tomography.py

```diff
@@ -0,0 +1,2089 @@
+            circ.add(gates.X(1))
+
+        elif k == 2:  # |0> |+>
+            circ = Circuit(nqubits, density_matrix=True)
+            circ.add(gates.H(1))
+
+        elif k == 3:  # |0> |y+>
+            circ = Circuit(nqubits, density_matrix=True)
+            circ.add(gates.H(1))
+            circ.add(gates.S(1))
+        # --------------------------------------------
+
+        elif k == 4:  # |1> |0>
+            circ = Circuit(nqubits, density_matrix=True)
+            circ.add(gates.X(0))
+
+        elif k == 5:  # |1> |1>
+            circ = Circuit(nqubits, density_matrix=True)
+            circ.add(gates.X(0))
+            circ.add(gates.X(1))
+
+        elif k == 6:  # |1> |+>
+            circ = Circuit(nqubits, density_matrix=True)
+            circ.add(gates.X(0))
+            circ.add(gates.H(1))
+
+        elif k == 7:  # |1> |y+>
+            circ = Circuit(nqubits, density_matrix=True)
+            circ.add(gates.X(0))
+            circ.add(gates.H(1))
+            circ.add(gates.S(1))
+
+        # --------------------------------------------
+
+        elif k == 8:  # |+> |0>
+            circ = Circuit(nqubits, density_matrix=True)
+            circ.add(gates.H(0))
+
+        elif k == 9:  # |+> |1>
+            circ = Circuit(nqubits, density_matrix=True)
+            circ.add(gates.H(0))
+            circ.add(gates.X(1))
+
+        elif k == 10:  # |+> |+>
+            circ = Circuit(nqubits, density_matrix=True)
+            circ.add(gates.H(0))
+            circ.add(gates.H(1))
+
+        elif k == 11:  # |+> |y+>
+            circ = Circuit(nqubits, density_matrix=True)
+            circ.add(gates.H(0))
+            circ.add(gates.H(1))
+            circ.add(gates.S(1))
+
+        # --------------------------------------------
+
+        elif k == 12:  # |y+> |0>
+            circ = Circuit(nqubits, density_matrix=True)
+            circ.add(gates.H(0))
+            circ.add(gates.S(0))
+
+        elif k == 13:  # |y+> |1>
+            circ = Circuit(nqubits, density_matrix=True)
+            circ.add(gates.H(0))
+            circ.add(gates.S(0))
+            circ.add(gates.X(1))
+
+        elif k == 14:  # |y+> |+>
+            circ = Circuit(nqubits, density_matrix=True)
+            circ.add(gates.H(0))
+            circ.add(gates.S(0))
+            circ.add(gates.H(1))
+
+        elif k == 15:  # |y+> |y+>
+            circ = Circuit(nqubits, density_matrix=True)
+            circ.add(gates.H(0))
+            circ.add(gates.S(0))
+            circ.add(gates.H(1))
+            circ.add(gates.S(1))
+
+        # --------------------------------------------
+
+        # If no class_qubit_gate
+        if class_qubit_gate is None:
+            pass
+
+        # If class_qubit_gate is present (with/without theta)
+        elif class_qubit_gate is not None:
+            if theta is None:
+                print(f"Gate set tomography for 2-qubit gate: {class_qubit_gate}.")
```

Comment:
Avoid printing things.

Reply:
Got it. But not even sparingly for the user to see what's going on in the code?

Reply:
No, the functions will be documented with the docstrings, thus, the user will only need to now what goes inside and comes outside.

Reply:
Sure! Along with the other issue above, I will print only when checking and remove the print in the next version of gate_set_tomography.py.